### PR TITLE
[BULK UPDATE] DocuTune - Fix build validation issues: docs-link-absolute (part 2)

### PR DESCRIPTION
**Pull request by alexbuckgit**
**Global effort to fix build validation errors**

The Microsoft Skilling team is fixing build validation errors and specific brand inconsistencies in Microsoft technical documentation for the rest of FY23 Q1. This will help address various accessibility, security, and usability issues. This PR was generated using DocuTune. It includes only targeted fixes and minor formatting adjustments.

DocuTune PRs improve quality of technical content by finding and automatically correcting a broad set of issues only where the correction is suitable. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

CorrelationId: 0df9c2d6-63f8-4d1e-8b57-b33611a06514
InitiativeId: docutune-build-validation-issues-2022-05
PointOfContact: Alex Buck

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

**Comment by buck1ey** [comment]
### :x: Organization membership check failed
Members of the MicrosoftDocs organization should use the private repo. Please move your changes to [https://github.com/MicrosoftDocs/Dynamics-365-Operations](https://github.com/MicrosoftDocs/Dynamics-365-Operations), and close this pull request.

File: articles/fin-ops-core/fin-ops/get-started/help-overview.md [hunk 1]
```diff
@@ -163,16 +163,12 @@ The following table lists our websites. Sites that have an asterisk (\*) next to
 
 | Site | Description |
 |------|-------------|
-| [Docs.microsoft.com](/dynamics365/) | This site hosts or links to all product documentation for Dynamics 365. |
+| [Microsoft Dynamics 365 documentation](/dynamics365/) | This site hosts or links to all product documentation for Dynamics 365. |
 | [Microsoft Learn](/learn/) | This site is the free Microsoft eLearning site. |
 | [Microsoft Dynamics Lifecycle Services (LCS)](https://lcs.dynamics.com/)\* | This site provides a cloud-based collaborative workspace that customers and partners can use to manage projects from pre-sales to implementation and operations. It's useful in all phases of an implementation. |
 | [Support blog](https://aka.ms/AXSupportBlog) | This site provides tips and tricks that are posted by the Support team. |
-| [Docs.microsoft.com/previous versions](/previous-versions/dynamics/) | This site hosts content from previous releases. |
+| [[](/previous) versions](/previous-versions/dynamics/) | This site hosts content from previous releases. |
```
Comment (annmariehitchcock) [inline articles/fin-ops-core/fin-ops/get-started/help-overview.md:170]: Previous versions should be in one set of brackets. [Previous versions](/previous-versions/dynamics/)

**Comment by annmariehitchcock** [comment]
#docutune-wait

**Comment by alexbuckgit** [comment]
Already handled via another repo.

#docutune-abandoned

**Pull request closed without merging**